helm_deploy/values-dev.yaml

```diff
@@ -1,9 +1,10 @@
 ---
 # Environment specific values, override helm_deploy/court-register/values.yaml
 generic-service:
-  replicaCount: 2
+  replicaCount: 1
   ingress:
-    host: offender-events-ui-dev.prison.service.justice.gov.uk
+    hosts:
```

Comment:
not a great fan of this change - the helm chart allows either so think just specifying the `host` seems simpler.

Reply:
ok, i was thinking "host" may be deprecated, but will revert 👍